test/screenshot/tab/index.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+import Tab, {TabProps} from '../../../packages/tab';
+// https://github.com/material-components/material-components-web-react/issues/513
+// @ts-ignore
+import MaterialIcon from '../../../packages/material-icon/index';
+import '../../../packages/tab-indicator/index.scss';
+import '../../../packages/tab/index.scss';
+import './index.scss';
+
+interface TabContentType {
+  num: number;
+}
+
+const TabContent: (args: TabContentType) => React.ReactElement<any> = ({
+  num, // eslint-disable-line react/prop-types
+}) => (
+  <React.Fragment>
+    <MaterialIcon className='mdc-tab__icon' icon='favorite' />
+    <span className='mdc-tab__text-label'>Tab {num}</span>
+  </React.Fragment>
+);
+
+interface TabsProps {
+  children: React.ReactNode;
+}
+
+const Tabs: (args: TabsProps) => JSX.Element = ({
+  children, // eslint-disable-line react/prop-types
+}) => {
+  return <div className='tabs'>{children}</div>;
+};
+
+interface TabsControllerProps extends TabProps {
+  tabContent: (num: number) => React.ReactElement<any>
+}
+
+interface TabsControllerState {
+  activeIndex: number;
+  previousActiveIndex: number;
+}
+
+class TabsController extends React.Component<TabsControllerProps, TabsControllerState> {
+  tabBoundingRects: {[n: number]: ClientRect} = {};
+
+  state = {activeIndex: 0, previousActiveIndex: 0};
+
+  tabInit = (tabEl: Tab | null, index: number) => {
```

Comment:
Can tabEl be optional instead of nullable?

Reply:
No it has to be nullable. But as @appsforartists pointed out it should be a React.Ref.